### DISCUSSION TITLE
How does one enable feature flags in an Istio installation with Helm

### DISCUSSION THREAD

**Discussion by matthewkrupnik** [istio/istio#51242] [Q&A] [answered]
I deployed Istio on EKS and pods can't seem to make any outbound connections. I came across github issue that seems to be relevant: https://github.com/istio/istio/pull/46800. It appears that there was a new feature flag added `ISTIO_ENABLE_IPV4_OUTBOUND_LISTENER_FOR_IPV6_CLUSTERS` to enable exactly the functionality that I need. However I can't seen to find any documentation whatsoever on where do I need to set this feature flag? Can anyone provide some guidance? I deployed Istio using Helm if that makes any difference.

**Comment by skripted-io** [comment]
Have a look here https://istio.io/latest/docs/reference/config/networking/proxy-config/

see "environmentVariables"

**Reply by howardjohn** [reply]
This is for the sidecar proxy env. For Istiod this would be `--set pilot.env.ISTIO_ENABLE_IPV4_OUTBOUND_LISTENER_FOR_IPV6_CLUSTERS=true`

**Comment by skripted-io** [comment, marked as answer]
Thanks @howardjohn, that indeed was the only way to make it work for me.

Still wondering why it's even necessary to set this. Can't Istio auto detect someone is running an IPv6-only cluster?

For helm/terraform users:

```
resource "helm_release" "istiod" {
  name             = "istiod"
  repository       = "https://istio-release.storage.googleapis.com/charts/"
  chart            = "istiod"
  namespace        = "istio-system"
  create_namespace = true
  cleanup_on_fail  = true

  set {
    name  = "pilot.env.ISTIO_ENABLE_IPV4_OUTBOUND_LISTENER_FOR_IPV6_CLUSTERS"
    value = "true"
  }
} 
```

**Reply by matthewkrupnik** [reply]
I eventually found the answer I was looking for on Istio slack and forgot to come back here and post. But for the next person looking, this is exactly what I ended up with since I use terraform to deploy helm chart. This is the way
